packages/react-ui/storybook/stories/widgetsUI/CategoryWidgetUI.stories.js

```diff
@@ -60,7 +60,7 @@ const LoadingTemplate = (args) => {
   );
 };
 
-const data = [...Array(30)].map((_, idx) => ({
+const data = [...Array(2000)].map((_, idx) => ({
```

Comment:
I've tested with up to 10K elements, and it works, but starting at that point, memory gets drained 👎 

I did some tests with `react-window`, and it supports millions of rows, so you probably know where the problem is at this point 😄